src/xdocs/writingjavadocchecks.xml.vm

```diff
@@ -558,11 +558,123 @@ JAVADOC -> JAVADOC [0:0]
         </td>
       </tr>
     </table>
-    </section>
 
-    <section name="Boolean flag indicating the presence of unclosed HTML tags">
```

Comment:
Why did we remove section?

Reply:
https://github.com/checkstyle/checkstyle/issues/3311#issuecomment-328093482

Reply:
Thanks I didn't see this.